.github/workflows/ci.yml

```diff
@@ -168,6 +172,9 @@ jobs:
     - name: Update pip, wheel, setuptools, build, twine
       run: |
         python -m pip install -U pip wheel setuptools build twine
+    - name: Test a workaround for #6600
+      if: startsWith(matrix.pyver, '3.11')
+      run: python -m pip install cython
```

Comment:
This doesn't seem necessary. Cythonize is called below depending on the `no-extensions` matrix factor. `cython` itself does get installed without an explicit separate command.
Instead, it should be updated in `requirements/cython.txt` and `requirements/constraints.txt`.